website/releases/0.9.0.md

```diff
@@ -0,0 +1,42 @@
rGithub":"https://github.com/binarylogic","dateString":"2020-04-09T00:00:00","description":"ACL is not always required when creating objects in GCP Cloud Storage","permalink":"/highlights/2020-04-09-make-acl-optional/","prNumbers":[2283],"release":"0.9.0","tags":["type: breaking change","domain: sinks","sink: gcp_cloud_storage"],"title":"ACL is now optional for the Google Cloud Storage sink","type":"breaking change"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-02-28T00:00:00","description":"Build Vector with select components","permalink":"/highlights/2020-02-28-custom-vector-builds/","prNumbers":[1924],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"À La Carte Custom Vector Builds","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-10T00:00:00","description":"Shed duplicate logs","permalink":"/highlights/2020-03-10-dedupe-transform/","prNumbers":[1848],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"New Dedupe Trasnform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-11T00:00:00","description":"Protect downstream metrics storages from runaway metrics tags","permalink":"/highlights/2020-03-11-tag-cardinality-limit-transform/","prNumbers":[1959],"release":"0.9.0","tags":["type: new feature","domain: transforms","transform: tag_cardinality_limit"],"title":"New Tag Cardinality Limit Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-31T00:00:00","description":"Filter and route your logs based on defined conditions","permalink":"/highlights/2020-03-31-filter-transform/","prNumbers":[2088],"release":"0.9.0","tags":["type: new feature","domain: transforms","transform: filter"],"title":"New Filter Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-01T00:00:00","description":"Easily rnrich your logs with Kubernetes metadata","permalink":"/highlights/2020-04-01-kubernetes-pod-metadata-transform/","prNumbers":[1888],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"New Kubernetes Pod Metdata Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-07T00:00:00","description":"The next iteration of our Lua transform.","permalink":"/highlights/2020-04-07-lua-transform-version-2/","prNumbers":[2126],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"Lua Transform v2","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-11T00:00:00","description":"We identified optimizations that net ~36% higher throughput","permalink":"/highlights/2020-04-11-overall-performance-increase/","prNumbers":[2295,2296],"release":"0.9.0","tags":["type: performance"],"title":"~36% Higher Throughput","type":"performance"}]}
```

Comment:
```suggestion
  items={[{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-23T00:00:00","description":"This brings the sink naming inline with consistent naming pattern","permalink":"/highlights/2020-03-23-rename-gcp_stackdriver_logging-sink-to-gcp_stackdriver_logs/","prNumbers":[2121],"release":"0.9.0","tags":["type: breaking change","domain: sinks","sink: splunk_hec"],"title":"The `gcp_stackdriver_logging` sink has been renamed to `gcp_stackdriver_logs`","type":"breaking change"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-05T00:00:00","description":"Compression is not supported by all Elasticsearch service providers","permalink":"/highlights/2020-04-05-default-compression-to-none/","prNumbers":[2219],"release":"0.9.0","tags":["type: breaking change","domain: sinks","sink: elasticsearch"],"title":"The `elasticsearch` sink `compression` option now defaults to `none`","type":"breaking change"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-09T00:00:00","description":"ACL is not always required when creating objects in GCP Cloud Storage","permalink":"/highlights/2020-04-09-make-acl-optional/","prNumbers":[2283],"release":"0.9.0","tags":["type: breaking change","domain: sinks","sink: gcp_cloud_storage"],"title":"ACL is now optional for the Google Cloud Storage sink","type":"breaking change"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-02-28T00:00:00","description":"Build Vector with select components","permalink":"/highlights/2020-02-28-custom-vector-builds/","prNumbers":[1924],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"À La Carte Custom Vector Builds","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-10T00:00:00","description":"Shed duplicate logs","permalink":"/highlights/2020-03-10-dedupe-transform/","prNumbers":[1848],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"New Dedupe Trasnform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-11T00:00:00","description":"Protect downstream metrics storages from runaway metrics tags","permalink":"/highlights/2020-03-11-tag-cardinality-limit-transform/","prNumbers":[1959],"release":"0.9.0","tags":["type: new feature","domain: transforms","transform: tag_cardinality_limit"],"title":"New Tag Cardinality Limit Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-31T00:00:00","description":"Filter and route your logs based on defined conditions","permalink":"/highlights/2020-03-31-filter-transform/","prNumbers":[2088],"release":"0.9.0","tags":["type: new feature","domain: transforms","transform: filter"],"title":"New Filter Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-01T00:00:00","description":"Easily rnrich your logs with Kubernetes metadata","permalink":"/highlights/2020-04-01-kubernetes-pod-metadata-transform/","prNumbers":[1888],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"New Kubernetes Pod Metdata Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-07T00:00:00","description":"The next iteration of our Lua transform.","permalink":"/highlights/2020-04-07-lua-transform-version-2/","prNumbers":[2126],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"Lua Transform v2","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-11T00:00:00","description":"We identified optimizations that net ~36% higher throughput","permalink":"/highlights/2020-04-11-overall-performance-increase/","prNumbers":[2295,2296],"release":"0.9.0","tags":["type: performance"],"title":"~36% Higher Throughput","type":"performance"}]}
```

It says stackdrive on the preview but it should be stackdriver?

Reply:
Thanks, fixed.